hs/src/Reach/Backend/JS.hs

```diff
@@ -627,6 +627,8 @@ jsExpr = \case
     mo' <- jsArg mo
     da' <- jsArg da
     return $ jsApply "stdlib.fromSome" [mo', da']
+  DLE_ContractFromAddress _at _addr -> do
+    return $ jsArray [jsString "None", "null"]
```

Comment:
This can't be the implementation. Either it needs to do the real thing or it needs to log the answer and then look it up